bindings/rust/src/lib.rs

```diff
@@ -160,6 +160,11 @@ impl From<ExecutionResult> for sys::FizzyExecutionResult {
 }
 
 impl Instance {
+    /// Get a read-only pointer to the module.
+    unsafe fn get_instance_module(&self) -> *const sys::FizzyModule {
```

Comment:
```suggestion
    unsafe fn get_module(&self) -> *const sys::FizzyModule {
```

Reply:
Good point. I tried to mirror the underlying C API names, but probably it makes no sense.